### PR TITLE
gh-113317: Argument Clinic: move C/Py identifier helpers into libclinic

### PR DESCRIPTION
**Pull request by erlend-aasland**
* Issue: gh-113317

### CODE REVIEW

File: Tools/clinic/libclinic/identifiers.py
Comment (erlend-aasland) [inline Tools/clinic/libclinic/identifiers.py:1]: @AlexWaygood: interested in painting the name for this new bikeshed? 🎨 🚲

**Review by sobolevn — APPROVED** [review]
Thanks! 👍

**Comment by erlend-aasland** [comment]
Thanks for the review(s)!

File: Tools/clinic/libclinic/identifiers.py
Comment (erlend-aasland) [reply Tools/clinic/libclinic/identifiers.py:1]: identifiers.py it is! 🖌️

**Pull request merged**